data/tlg0064/tlg001/tlg0064.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="lat">Periplus Hannonis</title>
+                <author>Hanno</author>
+                <editor xml:lang="lat">CAROLUS MULLERUS</editor>
+                <sponsor>HARVARD COLLEGE LIBRARY</sponsor>
```

Comment:
Starting from here and going down to the end of the titleStmt element, replace these elements with the following. This is the updated information for the project.

```
<funder>Harvard Library Arcadia Fund</funder>
<principal>Gregory Crane</principal>
<respStmt>
    <persName xml:id="DDD">Digital Divide Data</persName>
    <resp>Corrected and encoded the text</resp>
</respStmt>
<respStmt>
    <persName>Gregory Crane</persName>
    <resp>Editor-in-Chief, Perseus Digital Library</resp>
</respStmt>
<respStmt>
    <persName>Matt Munson</persName>
    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
</respStmt>
<respStmt>
    <persName>Annette Gessner</persName>
    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
</respStmt>
<respStmt>
    <persName>Thibault Clérice</persName>
    <resp>Lead Developer (University of Leipzig)</resp>
</respStmt>
<respStmt>
    <persName>Bruce Robertson</persName>
    <resp>Technical Advisor</resp>
</respStmt>
```
See issue #678 for more information on this section in the new files.